src/index.html

```diff
@@ -1,13 +1,237 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Eco Cosmetics</title>
+    <link
+      rel="icon"
+      href="./images/svg/logo.svg"
+      type="image/x-icon"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__body">
+
+    <header class="header" id="home">
+
+      <div class="header__container">
+        <div class="top-actions">
+          <a href="#menu" class="icon icon--menu"></a>
+          <a href="#" class="logo">
+            <img
+              src="./images/svg/logo.svg"
+              alt="Logo"
+              class="logo__img"
+            >
+          </a>
+          <a href="#bag" class="icon icon--shop"></a>
+        </div>
+
+        <nav class="page__nav nav">
+          <div class="nav__logo">
+            <a href="#" class="logo nav__logo">
+              <img
+                src="./images/svg/logo.svg"
+                alt="Logo"
+                class="logo__img"
+              >
+            </a>
+            <a href="#info" class="nav_description link">
+              witch of mountain winds
+            </a>
+          </div>
+          <div class="nav__content">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#shop" class="nav__link link">
+                  Shop
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a href="#contact" class="nav__link link">
+                  Contact
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a href="#about" class="nav__link link">
+                  About
+                </a>
+              </li>
+            </ul>
+          </div>
+
+          <a href="#bag" class="nav__bag">
+            <img
+              src="./images/svg/shop.svg"
+              alt="to shopping bag"
+              class="nav__svg"
+            >
+          </a>
+        </nav>
+
+        <div class="header__content">
+          <div class="grid grid--desktop grid--tablet">
+
+            <div class="grid__item grid__item--desktop-7-12 grid__item--tablet-1-6 grid__item-row">
+              <div class="header_img-cont">
+                <img
+                  src="./images/photos/header.png"
+                  alt="Products"
+                  class="header__img"
+                >
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--desktop-2-5 grid__item-row">
+              <div class="header__info">
+                <h1 class="header__title">
+                  Brand of
+                  <br>
+                  eco-cosmetics and magic candles
```

Comment:
You should't use here <br>. Also, use special symbol to disable wrapping word "eco-cosmetics"
```suggestion
                  Brand of eco&#8209;cosmetics and magic candles
```